src/Common/Nav.elm

```diff
@@ -0,0 +1,25 @@
+module Common.Nav exposing (navBar)
```

Comment:
@waiyaki Bruh, you need to sleep. 😄  I think there are two "NavBars", one for the landing page and one for the "app" page.